datachannel_go_test.go

```diff
@@ -195,3 +195,153 @@ func TestDataChannelParamters_Go(t *testing.T) {
 		assert.Equal(t, uint64(1500), dc.BufferedAmountLowThreshold(), "should match")
 	})
 }
+
+func TestDataChannelBufferedAmount(t *testing.T) {
```

Comment:
If we add Wasm support for `SetBufferedAmountLowThreshold` and `OnBufferedAmountLow`, we should be able to move this test to __datachannel_test.go__ (so that it will be run in both Wasm and vanilla Go environments). For now it belongs here.

Reply:
I updated https://github.com/pions/webrtc/issues/519 to mention this. It shouldn't be too hard to support in the Wasm bindings.

Reply:
Thanks @albrow !